jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -6358,21 +6375,23 @@
 						}
 					],
 					"request": {
-						"url": "{{url}}/v1.0/jvms/Postman-test-jvm2/jdk",
+						"url": "{{url}}/v1.0/jvms/{{JvmName2}}/jdk",
```

Comment:
I still don't see where the JvmName2 JVM has its JDK media changed to a different version before doing the upgrade. This test still looks like it's running the scenario where the JVM's JDK version hadn't changed and the upgrade JDK API was called - which is a valid test, but we're still not covering the core feature scenario.

Reply:
I am uploading a Setenv.bat file and then uploading a new JDK Media and then calling a update JVM REST API to update the newly uploaded JDK media to the JVM and finally calling the upgrade JDK REST call . This cover the positive workflow.